entity-framework/core/managing-schemas/migrations/applying.md

```diff
@@ -156,6 +159,76 @@ Note that this can be used to roll back to an earlier migration as well.
 
 For more information on applying migrations via the command-line tools, see the [EF Core tools reference](xref:core/cli/index).
 
+## Bundles
+
+> [!NOTE]
+> This feature was added in EF Core 6.0.
+
+Migration bundles are single-file executables than can be used to apply migrations to a database. They address some of the shortcomings of the SQL script and command-line tools:
+
+* Executing SQL scripts requires additional tools.
+* The transaction handling and continue-on-error behavior of these tools are inconsistent and sometimes unexpected.
+* Bundles can be generated as part of your CI process and executed later as part of your deployment process.
```

Comment:
This is true of SQL scripts though too, right? Not that we shouldn't mention it, but it doesn't seem like a reason to prefer bundles over SQL scripts per se...